### ISSUE TITLE
Removing App "System" let crash the simulator

### ISSUE THREAD

**Issue by wb7** [mozilla/r2d2b2g#847]
If you remove the app "System" over Settings -> App permissions -> System -> Uninstall, the simulator wont start a second time. The result is clear, but maybe you should remove this option, because it's a little bit crazy to remove the whole system.

**Comment by ccarruitero** [comment]
I agree with this, but this is b2g's feature. Maybe you can fill a bug in [bugzilla](https://bugzilla.mozilla.org/enter_bug.cgi)
